src/webgpu/shader/validation/parse/continue.spec.ts

```diff
@@ -0,0 +1,88 @@
+export const description = `Validation tests for continue`;
+
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { keysOf } from '../../../../common/util/data_tables.js';
+import { ShaderValidationTest } from '../shader_validation_test.js';
+
+export const g = makeTestGroup(ShaderValidationTest);
+
+const kTests = {
+  continue: {
+    src: 'continue;',
+    pass: false,
+  },
+  compound_continue: {
+    src: '{ continue; }',
+    pass: false,
+  },
+  loop_continue: {
+    src: 'loop { if false { break; } continue; }',
+    pass: true,
+  },
+  while_continue: {
+    src: 'while true { continue; }',
+    pass: true,
+  },
+  for_continue: {
+    src: 'for (;true;) { continue; }',
+    pass: true,
+  },
```

Comment:
`for(continue;;) {break;}`
`for(;continue;) {break;}`
`for(;;continue) {break;}`

Reply:
Done.